.github/workflows/build.yml

```diff
@@ -1,5 +1,5 @@
 name: Build/Test/Verify/Release
-on: [push, pull_request_target]
+on: [push, pull_request]
```

Comment:
Is this change security related?

Reply:
No, it was a misconfiguration. I misunderstood what the "context" meant. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target